mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -72,6 +72,8 @@
     <string name="browser_menu_your_library">Your Library</string>
     <!-- Browser menu toggle that requests a desktop site -->
     <string name="browser_menu_desktop_site">Desktop site</string>
+    <!-- Browser menu button that adds a top site to the home fragment -->
+    <string name="browser_menu_add_to_firefox_home">Add to Firefox home</string>
```

Comment:
Please update to placeholder here

Reply:
@sblatz 👀 last change to get in, before Delphine heads out.